src/cxx.c

```diff
@@ -1,10 +1,12 @@
 // SPDX-FileCopyrightText: 2013-2019 pancake <pancake@nopcode.org>
 // SPDX-License-Identifier: LGPL-3.0-only
 #include "demangler_util.h"
-#include "cxx/demangle.h"
+#include "borland.h"
```

Comment:
I think this change doesn't belong to this commit and should be moved to the next one?

Reply:
fixed.